corehq/apps/email/models.py

```diff
@@ -29,8 +36,21 @@ def plaintext_password(self):
             return b64_aes_decrypt(ciphertext)
         return self.password
 
+    @property
+    def plaintext_password_cbc(self):
```

Comment:
This name is confusing to me. Is the return value plaintext or CBC-encrpyted? I think it can be removed if Norman's suggestion of using an `aes-cbc` prefix is accepted.

Reply:
fair point, I ended up removing it in favor of Norman's suggestion.